src/scrape_up/hashnode/hashnode.py

```diff
@@ -44,48 +43,76 @@ def get_feed(self):
 
             blogs_data = {"blogs": []}
 
-            blogs = soup.find_all("div", class_="css-1s8wn94")
+            blogs = soup.find_all(
+                "div",
```

Comment:
This isn't working @Mihan786Chistie | `blogs` is returning an empty list.

Please check once

Reply:
Check it now @nikhil25803

Reply:
Still returning empty array @Mihan786Chistie